samples/bluetooth/hci_spi/README.rst

```diff
@@ -0,0 +1,19 @@
+Bluetooth: HCI SPI
+##################
+
+Overview
+********
+
+Expose Zephyr Bluetooth Controller support over SPI to another device/CPU using
+the Zephyr SPI HCI transport protocol (similar to BlueNRG).
+
+
+Requirements
+************
+
+* A board with SPI and BLE support
+
+Building and Running
+********************
+
+See :ref:`bluetooth setup section <bluetooth_setup>` for details.
```

Comment:
It seems that @mbolivar just copy-pasted existing README.rst from samples/bluetooth/hci_uart/ and samples/bluetooth/hci_usb/ . So, README in this patch is as good or as bad as READMEs for those samples. And indeed, they are pretty bad:

* https://www.zephyrproject.org/doc/samples/bluetooth/hci_uart/README.html
* https://www.zephyrproject.org/doc/samples/bluetooth/hci_usb/README.html

E.g.:

> Requirements
> * BlueZ running on the host, or
> * A board with BLE support

What does that "or" do there?